Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM public.ecr.aws/amazonlinux/amazonlinux:latest as builder
 # Fluent Bit version; update these for each release
 ENV FLB_VERSION 1.9.6
 # branch to pull parsers from in github.com/fluent/fluent-bit-docker-image
-ENV FLB_DOCKER_BRANCH 1.9
+ENV FLB_DOCKER_BRANCH master
```

Comment:
I think we should still use 1.8 branch since it seems to be the most recently updated here: https://github.com/fluent/fluent-bit-docker-image/tree/1.8

Master is not updated since 2021.